labs/dom/expand/expand.js

```diff
@@ -0,0 +1,3 @@
+(function() {
```

Comment:
This a total tangent thought but wondering if one of these should use a:

```js
document.addEventListener('DOMContentLoaded', fn)

// or 

window.addEventListener('load', fn)
```
For best practice // other ways to do things.

Reply:
Yeah probably good to be consistent and have the events bind after the file is loaded, I'll add em to all the Dom challanges later

Reply:
MMMM. I was thinking some variation with a short explanation. Like some just use IIFEs, some use event listeners etc.